docs/root/configuration/http/http_filters/cache_filter.rst

```diff
@@ -0,0 +1,46 @@
+.. _config_http_filters_cache:
+
+HTTP Cache Filter
+=================
+**Work in Progress--not ready for deployment**
```

Comment:
Can you put this in an `.. attention::` block?

Reply:
done